README.md

```diff
@@ -0,0 +1,7 @@
+# ubma-screenshots
+
+Here are some screenshots describing our software and the software
+we are supporting. By uploading the screenshots to this separate 
+directory it is possible to include a picture in markdown with
+just a link and thereby not making the code repository unneccassary
```

Comment:
Funny, I thought you meant the README was unnecessary. Now that your comment is a valid sed expression, I can understand it perfectly. :-)